# target x,y
relay_points:
  - [-2.0,  -1.0]
  - [0.5,  -1.0]
  - [2.0,  0.5]
  - [7.0, -1.5]
